Source/WebKit/UIProcess/mac/PageClientImplMac.mm

```diff
@@ -599,15 +599,15 @@ - (BOOL)_hostsLayersInWindowServer;
 {
     ASSERT(!layerTreeContext.isEmpty());
 
-    CALayer *renderLayer = [CALayer _web_renderLayerWithContextID:layerTreeContext.contextID];
+    CALayer *renderLayer = [CALayer _web_renderLayerWithContextID:layerTreeContext.contextID shouldPreserveFlip:false];
```

Comment:
`shouldPreserveFlip:false` -> `shouldPreserveFlip:NO` here and below

Reply:
Fixed.